sdk/search/search-documents/src/searchIndexingBufferedSender.ts

```diff
@@ -0,0 +1,351 @@
+import { SearchClient } from "./searchClient";
+import { IndexDocumentsBatch } from "./indexDocumentsBatch";
+import {
+  IndexDocumentsAction,
+  SearchIndexingBufferedSenderOptions,
+  SearchIndexingBufferedSenderUploadDocumentsOptions,
+  SearchIndexingBufferedSenderMergeDocumentsOptions,
+  SearchIndexingBufferedSenderMergeOrUploadDocumentsOptions,
+  SearchIndexingBufferedSenderDeleteDocumentsOptions,
+  SearchIndexingBufferedSenderFlushDocumentsOptions
+} from "./indexModels";
+import { IndexDocumentsResult } from "./generated/data/models";
+import { RestError, OperationOptions } from "@azure/core-http";
+import EventEmitter from "events";
+import { createSpan } from "./tracing";
+import { CanonicalCode } from "@opentelemetry/api";
+
+const DEFAULT_BATCH_SIZE: number = 1000;
```

Comment:
For this release, we are not allowing the user to modify these values. For next release, we will decide on what is the actual logic to be used. (Batch size vs Threshold Size, Simple Retry Count vs Retry Policy )